docs/angle_calculation.md

```diff
@@ -17,30 +19,20 @@ Here we explain the implementation of MOM6 angle calculation in regional-mom6, w
     2. The "x" component is the same addition of differences in latitude.
     3. Thus, given the same units, we can call arctan to get the angle in degrees
 
-
-## Problem
-MOM6 only calculates the angle at t-points. For boundary rotation, we need the angle at the boundary, which is q/u/v points. Because we need the points to the left, right, top, and bottom of the point, this method won't work for the boundary.
-
+6. **Additional step to apply to boundaries**
+Since the boundaries in a regional MOM6 model are on the q rather than t points, we need to expand the grid and calculate the angle at the boundary points. This is implemented in the `create_expanded_hgrid` function.
```

Comment:
```suggestion
Since the boundaries in a regional MOM6 model are on the `q` points and not on the `t` points, we need to expand the grid and calculate the angle at the boundary points. This is implemented in the `create_expanded_hgrid` function.
```